spec/models/user_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'spec_helper'
+
+describe User do
+  it { should validate_presence_of(:email) }
+  it { should validate_presence_of(:password) }
+  it { should validate_presence_of(:full_name) }
+  it { should validate_uniqueness_of(:email) }
+  it { should have_many(:queue_items).order("position") }
+
+  describe "#queued_item" do
```

Comment:
Is the method name just `queued_item` or `queued_item?`? the question mark in Ruby has significant meaning.